packages/coqide/coqide.8.12.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1-only"
+synopsis: "IDE of the Coq formal proof management system"
+description: """
+CoqIDE is a graphical user interface for interactive development
+of mathematical definitions, executable algorithms, and proofs of theorems
+using the Coq proof assistant.
+"""
+
+depends: [
+  "coq" {= version}
+  "lablgtk3-sourceview3"
```

Comment:
Did the problem with beta7 get fixed?

Reply:
I don't know, but the latest version 3.3.1 works. I don't think it is worth testing each intermediate version for incompatibilities, nor is it worth keeping a list of incompatible versions forever.

Reply:
In that case, just stick a constraint in to indicate the supported envelope.
```suggestion
  "lablgtk3-sourceview3" {>="3.1.1"}
```

Reply:
I used ` {>= "3.1.0"}` so that it at least gets rid of the 3.0beta versions, which seems sensible. This can always be modified later if you prefer.

Reply:
I have no idea what the future-compat policy is for these packages.